src/Workspaces/Core/MSBuild/MSBuild/MSBuildWorkspace.cs

```diff
@@ -277,6 +277,7 @@ public override bool CanApplyChange(ApplyChangesKind feature)
                 case ApplyChangesKind.RemoveProjectReference:
                 case ApplyChangesKind.AddAnalyzerReference:
                 case ApplyChangesKind.RemoveAnalyzerReference:
+                case ApplyChangesKind.ChangeProjectInfo:
                     return true;
```

Comment:
We shouldn't be saying this unless we actually implement the support for it. I'd be curious to see how difficult it actually is to make that work. The assumption is TryApplyChanges actually *should* be making the changes requested. Even though this PR as-is may fix #45710 let's see if we can avoid just creating a new bug of "oh, it returns true but still doesn't work".